packages/openauth/src/authorizer.ts

```diff
@@ -561,6 +561,64 @@ export function authorizer<
     },
   )
 
+  app.post(
+    "/invalidate",
```

Comment:
Do we want to support this under a `/revoke` operation to conform to the OAuth2 spec https://datatracker.ietf.org/doc/html/rfc7009
I'm wondering if we want to do this for just refresh tokens, or access tokens as well (although this is just a recommendation from the spec, not a requirement)